common/Environments/Models/ComputeSystem.cs

```diff
@@ -31,6 +34,48 @@ public class ComputeSystem
 
     public string DisplayName { get; private set; } = string.Empty;
 
+    [DllImport("user32.dll")]
+    [return: MarshalAs(UnmanagedType.Bool)]
+    private static extern bool AllowSetForegroundWindow(int dwProcessId);

```

Comment:
We can use CsWin32 here and use PInvoke. @krschau  I see you have an open issue for PInvoke usage in Dev Home. Are there any concerns there?

Reply:
Sergey made the same comment in another PR :)